packages/runtime-common/matrix-client.ts

```diff
@@ -51,6 +51,46 @@ export class MatrixClient {
     this.access = { accessToken, deviceId, userId };
   }
 
+  async getRooms() {
```

Comment:
consider extracting a `this.fetch` or similar method which encapsulates verifying the existence of the access token and setting the Authorization header. It could receive method and URL as arguments.

Reply:
Good suggestion, I adjusted that